pkg/cli/clients/management_test.go

```diff
@@ -1355,3 +1353,158 @@ func setCapture(ctx context.Context, response *http.Response) {
 func testCapture(ctx context.Context, capture **http.Response) context.Context {
 	return context.WithValue(ctx, holder{}, &holder{capture})
 }
+
+func Test_isResourceInEnvironment(t *testing.T) {
+	// Setup test cases
+	type testCase struct {
+		name          string
+		resource      generated.GenericResource
+		environmentID string
+		expected      bool
+	}
+
+	fullEnvID := "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/test-rg/providers/Applications.Core/environments/env1"
+	shortEnvID := "env1"
```

Comment:
In what scenario would the ID be represented as this short form instead of the full ID? Looking at json output from a valid environment, it usually looks like this: `"/planes/radius/local/resourcegroups/default/providers/Applications.Core/environments/default"`